controller.js

```diff
@@ -0,0 +1,111 @@
+
+const Controller = () => {
+    const moudl = moudle()
+    const vie = view()
+    const addingnewpost = (post) => {
+        moudl.addPost(post)
+        vie.renderPosts(moudl.getUpdatedPosts())
+    }
+    const addingnewcomment = (postid, comment) => {
```

Comment:
`postID`
keep your variables consistent in your code

Reply:
ok I didnt saw that